expts/neurips2023_configs/debug/config_large_gcn_debug.yaml

```diff
@@ -360,13 +360,16 @@ predictor:
 metrics:
   l1000_vcap: &classif_metrics
     - name: auroc
-      metric: auroc_ipu
+      metric: auroc
       num_classes: 5
       task: multiclass
+      target_to_int: True
+      target_nan_mask: -1000
+      ignore_index: -1000
```

Comment:
@DomInvivo  Had to use `ignore_index` for l1000_vcap and l1000_mcf7 as `target_nan_mask` gives nan. But for pcba `target_nan_mask` is used as you suggested.